hotel/tests/test_attendee_properties.py

```diff
@@ -25,6 +25,30 @@ def test_hotel_shifts_required_preshifts(monkeypatch):
     assert not Attendee().hotel_shifts_required
 
 
+@pytest.mark.parametrize('first,last,legal,expected', [
+    ('First', 'Last', None, 'Last'),
+    ('First', 'Last', 'First Last', 'Last'),
+    ('First', 'Last', 'First Middle Last', 'Last'),
+    ('CRAZY', 'Last', 'First Last', 'Last'),
+    ('CRAZY', 'CRAZY', 'First Last', 'Last'),
+    ('CRAZY', 'CRAZY', 'First Middle Last', 'Middle Last'),
+])
+def test_legal_last_name(first, last, legal, expected):
+    assert expected == Attendee(first_name=first, last_name=last, legal_name=legal).legal_last_name
+
+
+@pytest.mark.parametrize('first,last,legal,expected', [
+    ('First', 'Last', None, 'First'),
+    ('First', 'Last', 'First Last', 'First'),
+    ('First', 'Last', 'First Middle Last', 'First Middle'),
```

Comment:
You may have misunderstood this test. The expected result here (`First Middle`) is the `legal_first_name`, which is what we want.

If you look at the test for `legal_last_name` that uses the same inputs, you'll see that it returns `Last`, which may be what you're thinking of.